Makefile

```diff
@@ -67,7 +67,7 @@ dht: kademlia
 
 build: git_hooks
 	$(info Starting Build)
-	ulimit -s 65532 && (ulimit -n 10240 || true) && (ulimit -u 2128 || true) && cd src && $(WRAPSRC) env CODA_COMMIT_SHA1=$(GITLONGHASH) dune build --profile=$(DUNE_PROFILE)
```

Comment:
I think the `ulimit -u` is required for macos builds, but if CI passes, than I'm down with it (the mac default is 1418)

(this may be because we were accidentally building on `-j36` on CI before)

Reply:
I think this was an accident. I will add it back in for now, even though CI works. Unless you think we should limit our calls to ulimit.